src/components/ForestFire/ForestFireContainer.tsx

```diff
@@ -53,18 +122,34 @@ const ForestFireContainer = () => {
   const [stateDefinitions, setStateDefinitions] = useState<
     ReadonlyArray<ForestFireState>
   >(DEFAULT_STATE_DEFINITIONS);
+  const [isRunning, setIsRunning] = useState<boolean>(false);
+  const [currentIter, setCurrentIter] = useState(0);
   const [state, setState] = useState<ReadonlyArray<ReadonlyArray<number>>>(
-    DEFAULT_STATE
+    getInitialState(rows, columns, stateDefinitions)
   );
 
+  useEffect(() => {
+    setState(getInitialState(rows, columns, stateDefinitions));
+  }, [stateDefinitions, setState]);
+
+  if (isRunning) {
+    if (currentIter < MAX_ITER) {
+      setCurrentIter((x) => x + 1);
+      setState(getNextState(state, pLightning, pReviving, neighborhood));
+    } else {
+      setIsRunning(false);
+      setCurrentIter(0);
+    }
+  }
+
   return (
     <Container>
       <ModelHeader
         modelTitle='Forest Fire'
         modelDescription='
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-          sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-          Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
```

Comment:
What's this?

Reply:
idk.. i assumed it was prettier's doing...?